### ISSUE TITLE
'slideChange' event not being emited on init, slideChange listener not being called

### ISSUE THREAD

**Issue by pantcaser** [nolimits4web/swiper#3487]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 5.3.6.
* Platform/Target and Browser Versions: CORDOVA.
* Live Link or JSFiddle/Codepen or website with isssue: not provided

### What you did
Init Swiper with this options:
```
{
	on: {
		slideChange: mySlideChangeListener,
	}
}
```

### Expected Behavior
mySlideChangeListener being called

### Actual Behavior
mySlideChangeListener not being called (it was in v4.0.6)

**Comment by pantcaser** [comment]
Condition for emmiting event is (swiper.js:1797)

```
    if (swiper.initialized || swiper.runCallbacksOnInit) {
      swiper.emit('slideChange');
    }
```

Should be

```
    if (swiper.initialized || swiper.params.runCallbacksOnInit) {
      swiper.emit('slideChange');
    }
```

**Comment by nolimits4web** [comment]
Fixed with #3491
